### DISCUSSION TITLE
A separate server address for SFTP in node config

### DISCUSSION THREAD

**Discussion by nicholasveronico** [pelican-dev/panel#45] [Ideas]
This was a past suggestion created by another user on Pterodactyl's GitHub; however, this would be a great feature to have on Pelican!

https://github.com/pterodactyl/panel/issues/2483

**Comment by Doc94** [comment]
The other user here xd
I hope finally thia can be consider in any place...

**Comment by chums122** [comment]
This would be incredibly helpful!

**Comment by lancepioch** [comment]
Thanks, please follow #247
